### ISSUE TITLE
Inference on training data impossible??

### ISSUE THREAD

**Issue by foxale** [Azure/MachineLearningNotebooks#1849]
Hi, 

I want to detect potential model overfitting for Azure AutoML forecasting. Usually, I would tune a model on train and then evaluate separately on train and out-of-time test set. But it seems like I cannot do backtesting with forecasting models on train?? That doesn't seem right.

Am I missing something here?
